Makefile

```diff
@@ -96,6 +96,11 @@ test:
 test-examples:
 	python -m pytest -n auto --dist=loadfile -s -v ./examples/pytorch/
 
+# Run benchmark
+
+benchmark:
+	python3 benchmark/benchmark.py --config-dir benchmark/config --config-name generation --commit=diff backend.model=google/gemma-2b --multirun
+
```

Comment:
`--commit=diff` : this will run the benchmark on `main` and `current branch or head`.
(it won't work at this moment - only after PR being merged to `main` will work)

`backend.model=google/gemma-2b`: This is temporarily (for demonstration purpose)

The long-term would be having it like `backend.model=$(cat models_to_run.txt)` where `models_to_run.txt` would be prepared by a CI job step. 

And if no such file, it will run the default models (defined in `benchmark/benchmark.py` or )

Reply:
Thank you! Yes, step by step 🚀